devbox.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/0.12.0/.schema/devbox.schema.json",
+  "env": {
+    "DEVBOX_COREPACK_ENABLED": "true",
```

Comment:
Gonna remove corepack in a new PR. Sometimes it collides with the installed packages and then we get a wrong pnpm version.

Reply:
Hm. Removing it leads to another problem:
```
$ node -v
v18.19.0

$ pnpm -v
8.6.12

$ pnpm i
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

Your Node version is incompatible with "/Users/alex/work/silverback-template".

Expected version: ^18.19.0
Got: v18.17.1
```